.github/ISSUE_TEMPLATE/tracking_issue.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Tracking issue for challenge
+about: Create a new tracking issue for a challenge
+title: "[Tracking Challenge] "
+labels: Challenge
```

Comment:
So should we update the book instructions to only create an issue once the proposal has been accepted?

Reply:
Yes, I am in the middle of changing that as well

Reply:
I would think that's the right approach.

Reply:
The only problem here is that the "Challenge" label would be automatically added to issues that may not be a challenge.

Reply:
To resolve that, I can add a new generic issue template that would be for issues that have nothing to do with the challenges themselves. Would that work?

Reply:
I would prefer to eventually have a job that runs once a new challenge has been merged, and edits the tracking issue with the link to the book entry and add the "Challenge" label. But for now, we will have to manually keep things consistent.